angular/core/components/sidebar/sidebar.haml

```diff
@@ -0,0 +1,47 @@
+/ %md_sidenav.md-sidenav-left{md-component-id: "left", md_is_locked_open: "$mdMedia('sm-l')",  md-whiteframe: 4}
```

Comment:
It's commented out - it's not supposed to be included, sorry. We've had trouble extracting the sidebar into it's own haml file, so it's in angular.html.haml atm . @robguthrie, do we want to try and address this?